aws-apigateway-go-routes/handler/handler.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
We probably want to add copyright to all code files as in https://github.com/pulumi/examples/blob/master/azure-go-aks/main.go#L1